File: src/main/java/com/wxl/dyttcrawler/CrawlerDyttApplication.java
package com.wxl.dyttcrawler;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * Create by wuxingle on 2020/5/10
 * main
 */
@SpringBootApplication
public class CrawlerDyttApplication {

    public static void main(String[] args) {
        SpringApplication application = new SpringApplication(CrawlerDyttApplication.class);
        application.run(args);
    }

}

File: src/main/kotlin/com/wxl/dyttcrawler/CrawlerDyttApplication.kt
package com.wxl.dyttcrawler

/**
 * Create by wuxingle on 2021/9/11
 * main
 * -Dkotlinx.coroutines.debug
 */

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan("com.wxl.dyttcrawler.properties")
class CrawlerDyttApplication


fun main(args: Array<String>) {
    runApplication<CrawlerDyttApplication>(*args)
}

